<b>Genome informatics and machine learning based identification of antimicrobial resistance encoding features in <i>Escherichia coli</i> genomes representing globally prevalent lineages including high risk clonal complexes</b>

Sabiha Shaik<sup>1</sup>, Anuradha Singh<sup>1</sup>, Arya Suresh<sup>1</sup>, Niyaz Ahmed<sup>1</sup>


<sup>1</sup>Pathogen Biology Laboratory, Department of Biotechnology and Bioinformatics, University of Hyderabad, Hyderabad, India


Address correspondence to Niyaz Ahmed, niyaz.ahmed@uohyd.ac.in




<b>Abstract</b> 

<i>Escherichia coli (E. coli)</i>, a ubiquitous commensal/pathogenic member from the Enterobacteriaceae family, is responsible for high morbidity and mortality worldwide. With the rise in multidrug resistance (MDR) globally, <i>E. coli</i> has been listed as one of the Global Antimicrobial Resistance and Use Surveillance System (GLASS) priority pathogens. Understanding the resistance mechanisms and identifying the underlying genomic features responsible, is of utmost importance to combat the further spread of these multidrug resistant superbugs. While few of the globally prevalent sequence types (ST) of <i>E. coli</i> such as ST131, ST69, ST405, and ST648 have been previously reported to be highly virulent and multidrug resistant, there is still no clarity if certain ST lineages of <i>E. coli</i> have a greater tendency to acquire multidrug resistance. In this study, large-scale comparative genomics of 5653 genomes from 19 ST lineages revealed ST-wise prevalence patterns of genomic features such as antimicrobial resistance (AMR) genes/mutations, virulence genes, integrons and transposons. Interpretation of feature importance using Random Forest Classifier trained with 11988 genomic features from whole genome sequence data identified ST-specific/phylogroup-specific signature proteins majorly belonging to different protein superfamilies including toxin-antitoxin systems.

